src/nccl_ofi_rdma.c

```diff
@@ -3365,6 +3409,74 @@ static int send_comm_destroy(nccl_net_ofi_rdma_send_comm_t *s_comm)
 	return ret;
 }
 
+/**
+ * Process all finalizing s_comm's, and destroy if done
+ *
+ * Note: caller must own the comm_cleanup_list_lock when calling
+ * this function
+ */
+static int send_comm_process_all_finalizing(void)
```

Comment:
I'm not sure what this function does.  Does it block, does it not block, what is finalizing?